src/libraries/System.Formats.Tar/src/System/Formats/Tar/GnuTarEntry.cs

```diff
@@ -38,6 +38,11 @@ public GnuTarEntry(TarEntryType entryType, string entryName)
         /// <summary>
         /// Initializes a new <see cref="GnuTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the GNU format.
         /// </summary>
+        /// <param name="other">The <see cref="TarEntry"/> instance to convert to the GNU format.</param>
+        /// <exception cref="InvalidOperationException"><para><paramref name="other"/> is a <see cref="PaxGlobalExtendedAttributesTarEntry"/> instance.</para>
+        /// <para>-or-</para>
+        /// <para>The entry type of <paramref name="other"/> is not supported in GNU.</para></exception>
+        /// <exception cref="FormatException">The format of <paramref name="other"/> is invalid.</exception>
```

Comment:
is https://github.com/dotnet/runtime/blob/8755447eed2c1d49c6a250f35bef4019122791e6/src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHelpers.cs#L371 possible?

Reply:
Yep, via the `base` constructor call. I added it in line 42.